defender/datadog_forta_detection_bot_health/autotask-1/index.js

```diff
@@ -1,3 +1,6 @@
+const stackName = 'datadog_alerts_heat_map';
```

Comment:
Should this be `datadog_forta_detection_bot_health`?

Reply:
Yes, good catch!

Reply:
Fixed in 57275781ad708017d54256a956b81395df9fc4ff